assets/training/model_management/components/validation_trigger_import/spec.yaml

```diff
@@ -0,0 +1,178 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+name: validation_trigger_import
+display_name: Validation Trigger Import model
+description: Component for enabling validation of import pipeline.
+version: 0.0.1
+
+# Pipeline inputs
+inputs:
+  # pipeline specific compute
+  compute:
+    type: string
+    optional: true
+    default: serverless
+    description: Common compute for model download, MLflow conversion and registration. eg. provide 'FT-Cluster' if
+      your compute is named 'FT-Cluster'. Special characters like \ and ' are invalid in the parameter value.
+      If compute name is provided, instance_type field will be ignored and the respective cluster will be used
+
+  instance_type:
+    type: string
+    optional: true
+    default: STANDARD_NC6s_v3
+    description: Instance type to be used for the component in case of serverless compute, eg. STANDARD_NC6s_v3. 
+      The parameter compute must be set to 'serverless' for instance_type to be used
+
+  ## Inputs for download model
+  model_source:
+    type: string
+    description: Storage containers from where model will be sourced from
+    default: Huggingface
+    enum:
```

Comment:
Could we please check test coverage for these model sources?

Reply:
Sure, we can take this as seperate task.